Update ukraine.md (#2974)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/ukraine.md

* [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
* [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)

### Deadline for applications

> You can apply for teacher training courses starting in autumn 2023 until [September 2023](https://www.apply-for-teacher-training.service.gov.uk/provider/service-guidance/dates-and-deadlines).

Reviewer: ```suggestion
You can [apply for teacher training courses](https://www.gov.uk/apply-for-teacher-training) starting in autumn 2023 until September 2023.
```
For accessibility reasons, we should have more descriptive link text than 'September 2023', and looking at the page you've linked to, I'm not sure it's necessarily extra info users would need? Is it maybe enough just to give them the date applications close? (Especially because the link goes to Manage, which is a provider facing service rather than for candidates.)

> If you intend to [apply](https://www.gov.uk/apply-for-teacher-training), you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience until your teacher training course starts. 

Reviewer: ```suggestion
You could also consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience until your teacher training course starts. 
```
We should avoid using one word links for accessibility reasons, so I've moved the apply link to the above sentence instead.
